downloadMesheryCandidate.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/sh
+
+OSARCHITECTURE="$(uname -m)"
+OS="$(uname)"
+if [ "x${OS}" = "xDarwin" ] ; then
+  OSEXT="osx"
+else
+  OSEXT="linux"
+fi
+if [ "x${MESHERY_VERSION}" = "x" ] ; then
+  MESHERY_VERSION=$(curl -L -s https://api.github.com/repos/layer5io/meshery/releases | \
+                  grep tag_name | sed "s/ *\"tag_name\": *\"\\(.*\\)\",*/\\1/" | \
+                  grep -v "rc\.[0-9]$" | sort -t"." -k 1,1 -k 2,2 -k 3,3 -k 4,4 | tail -n 1)
+fi
+
+if [ "x${MESHERY_VERSION}" = "x" ] ; then
+  printf "Unable to get latest mesheryctl version. Set MESHERY_VERSION env var and re-run. For example: export MESHERY_VERSION=v0.1.6"
+  exit;
+fi
+
+
+#mesheryctl_0.1.6_Darwin_i386.zip
+#echo ${A:1}
+
+
+#NAME="mesheryctl-$MESHERY_VERSION"
+URL="https://github.com/layer5io/meshery/releases/download/${MESHERY_VERSION}/mesheryctl_${MESHERY_VERSION:1}_${OSEXT}_${OSARCHITECTURE}.zip"
+#URL="https://github.com/layer5io/meshery/releases/download/v0.1.6/mesheryctl_0.1.6_Darwin_i386.zip"
+#printf "Downloading %s from %s ..." "$NAME" "$URL"
+#curl -L "$URL" | tar xz
+
+curl -L ${URL} -o /tmp/meshery.zip
```

Comment:
@anandrkskd can we use the user's current working directory as the download location?  `/tmp` isn't available on MacOS by default nor on Windows.